app/views/case_logs/_tasklist.html.erb

```diff
@@ -6,6 +6,9 @@
           <%= section.label %>
         </span>
       </h2>
+      <% if section.description %>
+        <p class="govuk-body"><%= section.description.html_safe %></p>
```

Comment:
Should this not be slightly smaller/hint text coloured? Looks very prominent in the screenshot. (Gets back into own lane...)

Reply:
Hint text should be avoided, where possible (the dark grey colour has a known issues), and anyway, we want users to see this message.